client/src/main/scala/skuber/Toleration.scala

```diff
@@ -0,0 +1,31 @@
+package skuber
```

Comment:
It is a matter of style really but as tolerations are tied to pods (and used in pod specs only) - I would prefer to define the Toleration type in the Pod scope (i.e. within the Pod companion object) similar to e.g. Pod.Affinity, Pod.Condition. 
Does this seem reasonable?

Reply:
I guess it is, just Pod is kinda big already.